examples/bonxai_map_playground.cpp

```diff
@@ -1,26 +1,35 @@
 #include <octomap/OcTree.h>
-#include "bonxai_map/pcl_utils.hpp"
-#include "bonxai_map/pointcloud.hpp"
 #include "pcl/point_types.h"
 
+#include "bonxai_map/probabilistic_map.hpp"
+#include "bonxai_map/pcl_utils.hpp"
+
 int main()
 {
   using namespace Bonxai;
 
-  Point3D testA = Eigen::Vector3d(1,2,3);
-//  Point3D testB = pcl::PointXYZ(1,2,3);
+  // constants
+  const double voxel_res = 0.02;
+  const double inv_resolution = 1.0 / voxel_res;
+  const Point3D test_point = Eigen::Vector3d{0.25, 0.12, 1.6};
 
-  std::vector<CoordT> points;
-  ComputeRay({0, 0, 0}, {0.25, 0.12, 1.6}, 0.02, points);
-  WritePointsFromPCD("point_to_point.pcd", points);
+  // run bonxai
+  std::vector<Bonxai::CoordT> bonxai_ray;
+  const auto coord = Bonxai::PosToCoord(test_point, inv_resolution);
+  ComputeRay({0, 0, 0}, coord, bonxai_ray);
+  std::cout << "points bonxai: " << bonxai_ray.size() << std::endl;
 
-  std::cout << "points bonxai: " << points.size() << std::endl;
+  std::vector<Point3D> points;
+  for(const auto& ray_coord : bonxai_ray){
+    points.push_back(CoordToPos(ray_coord, voxel_res));
+  }
+  WritePointsFromPCD("point_to_point.pcd", points);
```

Comment:
Side note, is it expected that a ray drawn by Bonxai should differ from Octree in length significantly?
At this resolution, I am seeing a difference of several cells :thinking: 
```
points bonxai: 80
points octree: 97
```

Reply:
the rasterization algorithm is probably different, this explain the tiny difference between the two algorithms